ocaml/xapi/xapi_bond.ml

```diff
@@ -431,6 +431,21 @@ let create ~__context ~network ~members ~mAC ~mode ~properties =
             Db.PIF.set_disallow_unplug ~__context ~self:pif ~value:false)
           members
       end;
+
+      (* If the host has a cluster_host AND the cluster_host's PIF is a member, update the PIF *)
+      begin match Xapi_clustering.find_cluster_host ~__context ~host with
```

Comment:
To declutter this already very long function, and reduce duplication with the destroy code below, please put (some of) this block into a new function, e.g. `move_cluster_pif`.

Reply:
For the other things that potentially need moving, information is gathered at the top of the function. It would make sense to do the same for this new case.